### ISSUE TITLE
Multi-user / multi-tab editing

### ISSUE THREAD

**Issue by yankiara** [soflyy/oxygen-bugs-and-features#1066]
**Describe the feature you'd like to see included in Oxygen.**
We absolutely need an edit mode where stylesheets and global settings are not saved, because at the moment, the editor cannot be safely used by two persons at the same time, even on different pages!

It is already pretty annoying to keep only one editor tab open during development phase, but it is unbearable for the client, who can't have two persons/employees edit different contents at the same time or one person and me as a dev/webmaster.

I really like that global things are available in the editor, for I can see live changes when editing them, but systematically saving them with content is a serious drawback. (Not to mention it really goes against content/style separation concept.)

Thx in advance.

**What are the use cases for this feature?**
Editing different contents at the same time.

**Examples of this feature or functionality.**
Any other page builder out there.

**Comment by benfavre** [comment]
This 100%

**Comment by KarzanTT** [comment]
+1000

**Comment by shoelaced** [comment]
Could global styles possibly update in real time across open editors via ajax, or pop up a notice when someone saves style changes, with a button to reload the styles? Only thing I suppose is that changes to global styles would have to be saved immediately, independent of the blue button save, otherwise things might still get overwritten as you're working, but obviously that would cause issues on a production site... maybe some kind of temporary cache that would hold the changes until someone clicks the blue button? 🤷🏽‍♂️ Anyway, definitely looking forward to the edit locking feature.

**Comment by benfavre** [comment]
Just don't save them if they are not modified as a start

**Comment by YammYcoding** [comment]
the global stuff could sync to each other everytime anyone edit something or save

**Comment by LauGau** [comment]
It’s a serious problem.
I would love to be able to open multiple pages with Oxy editor and copy/paste blocks or sections between them... 
I also want to be able to collaborate and not stress that a client open another editor thinking that no one else is editing.

**Comment by christianmagill** [comment]
I feel this should be the highest priority. This one issue is what makes Oxygen a non-starter for serious projects. Any organization with a moderate amount of content is goign to require more than one user be able to edit at a time.

**Comment by avocadesign** [comment]
+1000

**Comment by israeljernigan** [comment]
This is...like...why isn't this a thing???

**Comment by lindseynwmn** [comment]
Really need this to be able to scale!

**Comment by janblythe** [comment]
+1 this is driving us a bit nuts in a three person team - we have to constantly "checkout" websites in a slack channel to do any serious design work. Means less dedicated effort can be applied to a site at any one time.

**Comment by michacassola** [comment]
This is really important. Please make this a priority.

**Comment by WildPress** [comment]
Larger sites often have multiple developers and content editors working on them. Only allowing one Oxygen instance globally is really a big hinderance. 

I've only just started using Oxygen for projects, but this is an issue I know clients will be frustrated about. 

I don't want to go back to using other page builders (which all seem to handle this issue without any qualms).

Please consider making this a priority, as I believe it's holding Oxygen back from being used on bigger/more active websites.

**Comment by ms971** [comment]
> Larger sites often have multiple developers and content editors working on them. Only allowing one Oxygen instance globally is really a big hinderance.
Try https://collaboration.docs.oxyrealm.com - it solved the issue for us.

**Comment by oxyrealm-dev** [comment]
Hi
we have developed a solution for "Multi-user / multi-tab editing" issue.

[https://collaboration.docs.oxyrealm.com](https://collaboration.docs.oxyrealm.com)

![hqdefault](https://user-images.githubusercontent.com/79907810/128457582-46d3870d-f2e9-4179-a1e5-b433361e276c.jpg)

**Comment by WildPress** [comment]
Hey @oxyrealm-dev thank you for taking the time to develop a much-needed feature. I'm just wondering, have you spoken to the Oxygen devs about integrating this as a core feature as part of the plugin?

**Comment by elmichalos** [comment]
Much needed

**Comment by jonathan-cfmp** [comment]
This deserves a native solution to make oxygen viable for teams and agencies.

**Comment by elmichalos** [comment]
I think we should RaTHer USe a BrEAkdANcE

**Comment by JBCarroll** [comment]
This definitely requires full attention.

**Comment by Jens180980** [comment]
How can this, almost 3 years after issue was posted, still not be a core feature? Relying on a third-party paid plugin is not the way forward! This is such a turn off for any seriuos web company, and such a shame, since Oxygen in my opinion is the best page builder out there. Come on, Soflyy, you can do it.

**Comment by Everhardt-deatheater** [comment]
Love the builder, however enterprise clients, or really any client that has a team working on multiple pages will not migrate to Oxygen until multiple authors can edit multiple pages at the same time. It is the deal-breaker and I love Oxygen, every developer that I have introduced it to loves using it until I get that meeting to discuss this issue where they have to wait in line to edit their page. Please, this is an awesome builder but can not enter the multi user space until we can all edit our pages at the same time.

**Comment by Clag75** [comment]
This feature should be added by now. Please vote for this to be in the next release of Oxygen. Multi-User in the backend is a must for design teams and for clients to edit their sites after going live. Ever other Wordpress theme or theme builder allows this.

**Comment by AdiNadi1000** [comment]
+1

**Comment by greggtilston** [comment]
I can't +1,000,000,000 this enough.

**Comment by Everhardt-deatheater** [comment]
How do I request more of this?

**Comment by Everhardt-deatheater** [comment]
> This feature should be added by now. Please vote for this to be in the next release of Oxygen. Multi-User in the backend is a must for design teams and for clients to edit their sites after going live. Ever other Wordpress theme or theme builder allows this.

I R an Idiot, how do I vote for this?

**Comment by Everhardt-deatheater** [comment]
Agreed. It is an enterprise level builder that is keeping itself out of the
enterprise space with this one issue.

**Comment by supersuryaansh** [comment]
This 1000x

**Comment by Clag75** [comment]
Elijah Mills @Spellhammer, PLEASE ADD THIS FEATURE! We would be able to create more sites if our clients could edit at the same time as another user. Just lock the page/post so two users can't edit at the same time like other builders. This has been a request for some time and I'm hoping this is coming in the near future. Love Oxygen but not liking some of the limitations and slow feature releases. We will switch to monthly or annual fees if it will speed up some production on the builder.

**Comment by Mamadzhanov** [comment]
Can't believe this basic, but crucial functionality was not yet implemented. Since the team will focus on Oxy 6, this may be never be implemented. Sad...
